custom_components/plugwise/usb.py

```diff
@@ -168,23 +216,42 @@ async def async_will_remove_from_hass(self):
     @property
     def available(self):
         """Return the availability of this entity."""
-        return getattr(self._node, USB_SENSORS[AVAILABLE_SENSOR_ID][ATTR_STATE])()
+        return getattr(self._node, STICK_API[USB_AVAILABLE_ID][ATTR_STATE])
```

Comment:
Isn't this a very complex way to pass along the string "available"?
Why not define a constant: AVAILABLE = "available"?  Or use AVAILABLE_SENSOR_ID?

Reply:
You're right! I'll simplify it to just get the "available" attribute.

Reply:
Fixed in [62a5a7e](https://github.com/plugwise/plugwise-beta/pull/147/commits/62a5a7e1fbe0362c68033f817034722de1962094)